src/main/kotlin/snyk/advisor/api/AdvisorApiClient.kt

```diff
@@ -13,7 +16,27 @@ class AdvisorApiClient private constructor(
     private lateinit var retrofit: Retrofit
     private lateinit var scoreServiceEndpoint: ScoreService
 
-    fun scoreService(): ScoreService {
+    fun getPackagesInfo(packageManager: AdvisorPackageManager, packageNames: List<String>): List<PackageInfo>? {
+        try {
+            log.debug("Executing request to Advisor api")
```

Comment:
Do we need this line? It will not very helpful by debugging or issue investigation. `HttpLoggingInterceptor` will log requests and response.

Reply:
HttpLoggingInterceptor is not used yet. Also even if it will, there will be only warn events. We might consider to review such logging after start usage of HttpLoggingInterceptor.